packages/amplication-server/src/core/notifications/modules/inApp/inAppNotification.service.ts

```diff
@@ -0,0 +1,38 @@
+import { Injectable, Inject, BadRequestException } from '@nestjs/common';
+import { IInAppNotification } from '../../contracts/inAppNotification.interface';
+import { Novu } from '@novu/node';
+import { WINSTON_MODULE_PROVIDER } from 'nest-winston';
+import { Logger } from 'winston';
+
+@Injectable()
+export class InAppNotificationService implements IInAppNotification {
+  constructor(
+    @Inject(WINSTON_MODULE_PROVIDER) private readonly logger: Logger
+  ) {}
+
+  async pushNotification(
+    notificationName: string,
+    userId: string,
+    payload: { [key: string]: any }
+  ): Promise<void> {
+    const novu = new Novu(process.env.NOVU_API_KEY);
```

Comment:
This also can be initiated one-time aswell. But I don't believe it will create an overhead for you having it here :)